base/applications/osk/settings.h

```diff
@@ -0,0 +1,17 @@
+﻿/*
+ * PROJECT:         ReactOS On-Screen Keyboard
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         Settings header file with function prototypes
+ * COPYRIGHT:       Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+#ifndef SETTINGS_OSK_H
+#define SETTINGS_OSK_H
+
+#include "osk.h"
```

Comment:
This is usually a bad practice to include other headers within a header file (unless it's the precompiled header).
This include should be in settings.c, as :
```c
#include "osk.h"
#include "settings.h"
```
or alternatively, you can `#include "settings.h"` inside osk.h; this will make its functions available in all the source files of osk.